frontend/package.json

```diff
@@ -14,13 +14,12 @@
     "enzyme-adapter-react-16": "^1.1.1",
     "express": "^4.16.3",
     "express-cache-controller": "^1.1.0",
-    "filepond": "^3.5.0",
+    "filepond": "4.3.0",
```

Comment:
Same comment about the package-lock here. Just make sure they're in sync 😄

Reply:
Thanks 👍